crates/astria-composer/src/searcher/collector.rs

```diff
@@ -123,13 +123,18 @@ impl Collector {
                 .send_timeout(
                     Transaction {
                         chain_id: chain_id.clone(),
-                        inner: tx,
+                        inner: tx.clone(),
                     },
                     Duration::from_millis(500),
                 )
                 .await
             {
-                Ok(()) => {}
+                Ok(()) => {
+                    info!(
+                        transaction.hash = %tx.hash,
```

Comment:
define `let hash = tx.hash` above the `match searcher_channel` line, so you don't have to `clone()` the `tx`

Reply:
would it make sense to just have the whole log before the match?

This is only logged if sending to the searcher is succesful, some txs might not be logged.

Reply:
yeah my bad here, fixed